test/extensions/transport_sockets/tls/cert_validator/spiffe/BUILD

```diff
@@ -19,6 +24,7 @@ envoy_extension_cc_test(
     ],
     data = [
         "//test/common/tls/test_data:certs",
+        "//test/extensions/transport_sockets/tls/cert_validator/spiffe:trust_bundles",
```

Comment:
Seems you only move the generated file to this directory. Sorry for the chaos, maybe keep your previous design is better if we cannot split all these out from the `tls/test_data` cleanly.